cmake/modules/RootCPack.cmake

```diff
@@ -126,6 +126,11 @@ else()
   set(CPACK_SOURCE_GENERATOR "TGZ;TBZ2")
 endif()
 
+#----------------------------------------------------------------------------------------------------
+# Execute pre packaging script
+#
+set(CPACK_INSTALL_SCRIPTS ${CMAKE_SOURCE_DIR}/cmake/modules/CPackREADME.cmake)
```

Comment:
What's the motivation of creating a new file for these few lines - that are only needed for `APPLE`?

Reply:
I think deferring the conversion from markdown to HTML is important and the only way I found to do that was to use `CPACK_INSTALL_SCRIPTS` which can only take `.cmake` files as argument. (I tried passing a function as an argument but was unable to)

Reply:
Thanks, can you add this to the file's documentation comment?

Reply:
I documented it like this - https://github.com/root-project/root/pull/5786/files#diff-2214ec3366e6aa1f30a40d035b2b743cR132-R133

Reply:
Thanks!